chromium_src/chrome/installer/mini_installer/mini_installer.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2019 The Brave Authors
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_INSTALLER_MINI_INSTALLER_MINI_INSTALLER_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_INSTALLER_MINI_INSTALLER_MINI_INSTALLER_H_
+
+#define StandardReferralCodeLen 6
+
+#include "../../../../../chrome/installer/mini_installer/mini_installer.h"
+
+namespace mini_installer {
+typedef StackString<128> ReferralCodeString;
+
+// Populates |referral_code| with a Brave referral code if one is
+// present in the installer filename. This may be a standard or an
+// extended referral code.
+bool ParseReferralCode(const wchar_t* installer_filename,
+                       ReferralCodeString& referral_code);
+
+} // namespace mini_installer
+#endif
```

Comment:
Nice cleanup! 👍 
Nit: Lint will catch if `#endif`  doesn't have comments like below :)
`#endif  // BRAVE_CHROMIUM_SRC_CHROME_INSTALLER_MINI_INSTALLER_MINI_INSTALLER_H_`

Reply:
My most recent change minimizes the patch file much more :)